plugins/tiles/src/main/java/org/apache/struts2/tiles/StrutsTilesInitializer.java

```diff
@@ -30,12 +30,22 @@
 
 public class StrutsTilesInitializer extends AbstractTilesInitializer {
 
+    public static final String STRUTS_TILES_ALTERNATE_FILERESOURCEHANDLING = "struts.tiles.alternatefileresourcehandling";
```

Comment:
I would use a camel case version here as more user friendly, eg.:
```
struts.tiles.alternateFileRsourceHandling
```

Reply:
Thanks for pointing that out.  It has been changed to camel case in an updated commit (and the JIRA comment now references the new value as well).